karma_config/karma.conf.js

```diff
@@ -8,6 +8,7 @@ var path = require('path');
 webpack_config.plugins.push(new RewirePlugin());
 webpack_config.devtool = '#inline-source-map';
 webpack_config.resolve.alias['kolibri'] = path.resolve(__dirname, './kolibriGlobalMock');
+webpack_config.resolve.alias['vue-test'] = path.resolve(__dirname, './vueLocal');
```

Comment:
Make a special alias to bundle up all our additional plugins for vue, like vue-router, vuex, and vue-intl.